DatadogSessionReplay/Sources/Feature/RequestBuilder/RequestBuilder.swift

```diff
@@ -12,18 +12,24 @@ internal struct RequestBuilder: FeatureRequestBuilder {
 
     /// Custom URL for uploading data to.
     let customUploadURL: URL?
+    /// Sends telemetry through sdk core.
+    let telemetry: Telemetry
+    /// Builds multipart form for request's body.
+    var multipartBuilder: MultipartFormDataBuilder = MultipartFormData(boundary: UUID())
 
     func request(for events: [Data], with context: DatadogContext) throws -> URLRequest {
-        let source = SRSegment.Source(rawValue: context.source) ?? .ios // TODO: RUMM-2410 Send telemetry on `?? .ios`
```

Comment:
nice to see it's been already acknowledged in the past. All features will benefit from it 🙌